parser-service/src/main/java/org/mule/parser/service/strategy/RamlParsingStrategy.java

```diff
@@ -36,23 +42,37 @@ public ParseResult parse(ApiReference ref) {
     }
   }
 
+  private List<String> getReferences(ApiReference ref) {
+    ParseResult result = parsingStrategy.parse(ref);
```

Comment:
Default behavior is to run AMF in the first place, so at this point is likely that AMF parsing has already been done. Can we use that first parse to store refs? (Maybe having a global cache for references)